modules/version.rb

```diff
@@ -1 +1,8 @@
-$version = 4.3
+# frozen_string_literal: true
+
+module YuukiBot
+  class << self
+    attr_reader :version
+  end
+  @version = 4.3
```

Comment:
All this work deserves a version bump!
```suggestion
  @version = 4.4
```